vtex-apps/quickorder
# [QUICKORDER-10] fixed a bug where changing sellers would not change item status

#### How to test it? 
On https://weiworkspace--sandboxusdev.myvtex.com/quickorder

In Copy/paste, use this
880030a,11
880010a,98
Change the seller for row item **880030a** and see the item status update
![image](https://user-images.githubusercontent.com/32426660/153265377-747e2455-a124-47a5-a532-0ab800b82184.png)
![image](https://user-images.githubusercontent.com/32426660/153265393-4623d034-17d3-4b5b-aeff-b8229f675cbd.png)
